python/lsst/qserv/tests/mysqlDbLoader.py

```diff
@@ -49,7 +49,8 @@ def __init__(self, config,
                                              data_reader,
                                              db_name,
                                              multi_node,
-                                             out_dirname)
+                                             out_dirname,
+                                             [])
```

Comment:
using a named argument here would make the purpose of the empty list a little more clear (as you did in qservDbLoader.py 👏)